# v2.12.7

* `call_in_future` callable will now return the Future
* added `ExecutorWrapper`
* added `Future.on_success` and `Future.on_failure`

